admin-dev/themes/new-theme/scss/pages/_improve.scss

```diff
@@ -3,3 +3,84 @@
     width: 100%;
   }
 }
+
+#carrier_shipping_location_and_costs {
```

Comment:
Probably better to use `.js-multiple-zone-choice` instead of this ID 🤔

Reply:
IDs are okay for styling, in fact, they are the fastest way to target the element.

The point of using js- prefix is to differentiate classes used for JS manipulation.

Reply:
it breaks the css if I implement this